database/utils/permissions.js

```diff
@@ -0,0 +1,54 @@
+const api = require('../../src/api/v1/api.json')
+
+const permissionsForRoles = Object.values(api.routes).reduce((acc, el) => {
```

Comment:
Can we format this better please, it's quite hard to read. Better variable names would be great too (`el`, `el2`, `acc2` 🤷🏼‍♂️).